docs/operators/s3outputmodule.md

```diff
@@ -0,0 +1,176 @@
+S3OutputModule
+==============
+
+### About Amazon S3
+-------------------
+
+**Amazon S3 (Simple Storage Service)** is an object storage system with a web service interface and retrieve any amount of data at any time from anywhere on the web, offered by Amazon Web Services.
+
+### About S3 Output Module
+--------------------------
+
+Purpose of S3Output module is to upload files/directories into an Amazon S3 bucket using the multipart upload feature.
+
+S3Output module is **fault-tolerant**, **statically/dynamically partitionable** and has **exactly once** semantics.
+
+Module class is **S3OutputModule** located in the package **org.apache.apex.malhar.lib.fs.s3**; please refer to [github URL](https://github.com/apache/apex-malhar/blob/master/library/src/main/java/org/apache/apex/malhar/lib/fs/s3/S3OutputModule.java).
+
+Following diagram illustrates the DAG in this module:
+
+![](images/s3output/s3outputmodule.png)
+
+### About the operators in S3OutputModule
+-----------------------------------------
+
+-   ***S3InitateFileUploadOperator***
+    -   Initiate the upload for the file only if the number of blocks for a file is > 1. Amazon S3 returns a response with an upload ID, which is the unique identifier for the multipart upload. This upload ID must include whenever upload the blocks, complete an upload.
```

Comment:
is > 1 => is greater than 1
Amazon S3 => S3 {seems a bit redundant to say Amazon S3 in each occurrence since we've already described S3 as being an Amazon offering
early in this document}
The last sentence "This upload ID ..." is not clear -- not sure what it is trying to say.